src/components/ActionsToolbar/__stories__/ActionsToolbar.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {noop} from 'lodash';
+import cn from 'bem-cn-lite';
+
+import {Meta, Story} from '@storybook/react';
+import {CogIcon} from '../../../icons/CogIcon';
```

Comment:
Let's use icon from `@gravity-ui/icons` pack (we will change this icon in all project further)